test/ios-project-service.ts

```diff
@@ -199,16 +199,21 @@ describe("iOSProjectService", () => {
 				}
 			};
 		}
-		it("by default exports xcodearchive to platforms/ios/build/archive/<projname>.xcarchive", async () => {
-			let setup = await setupArchive();
-			await setup.run();
-			setup.assert();
-		});
-		it("can pass archivePath to xcodebuild -archivePath", async () => {
-			let setup = await setupArchive({ archivePath: "myarchive.xcarchive" });
-			await setup.run();
-			setup.assert();
-		});
+
+		if (require("os").platform() !== "darwin") {
```

Comment:
in many places ;) 
These are tests, so I'll live with it for the moment. In the future we may introduce a property of `it` and `define` and handle it automatically.